test/integration/Utils.kt

```diff
@@ -15,14 +15,52 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  *
  */
+@file:Suppress("INVISIBLE_REFERENCE", "INVISIBLE_MEMBER")
 
 package com.vaticle.typedb.studio.test.integration
 
+import androidx.compose.ui.test.assertAll
+import androidx.compose.ui.test.hasClickAction
 import androidx.compose.ui.test.junit4.ComposeContentTestRule
+import androidx.compose.ui.test.onAllNodesWithText
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import com.vaticle.typedb.client.api.TypeDBSession
+import com.vaticle.typedb.client.api.TypeDBTransaction
+import com.vaticle.typedb.studio.framework.material.Icon
+import com.vaticle.typedb.studio.state.StudioState
+import com.vaticle.typedb.studio.state.common.util.Label
+import kotlinx.coroutines.delay
 import kotlinx.coroutines.runBlocking
+import java.io.File
 import java.nio.charset.StandardCharsets
 import java.nio.file.Files
+import java.nio.file.Path
 import java.nio.file.Paths
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+
+val SAVE_ICON_STRING = Icon.Code.FLOPPY_DISK.unicode
+val PLUS_ICON_STRING = Icon.Code.PLUS.unicode
+val PLAY_ICON_STRING = Icon.Code.PLAY.unicode
+val CHECK_ICON_STRING = Icon.Code.CHECK.unicode
+val ROLLBACK_ICON_STRING = Icon.Code.ROTATE_LEFT.unicode
+val CHEVRON_UP_ICON_STRING = Icon.Code.CHEVRON_UP.unicode
+val DOUBLE_CHEVRON_DOWN_ICON_STRING = Icon.Code.CHEVRONS_DOWN.unicode
+val DOUBLE_CHEVRON_UP_ICON_STRING = Icon.Code.CHEVRONS_UP.unicode
+val ARROW_FROM_SQUARE_ICON_STRING = Icon.Code.ARROW_UP_RIGHT_FROM_SQUARE.unicode
+val REFRESH_ICON_STRING = Icon.Code.ROTATE.unicode
+val XMARK_ICON_STRING = Icon.Code.XMARK.unicode
```

Comment:
A lot of these icon strings are used in multiple places, so we define them in the Utils file to avoid repetition. 

I've renamed a few of these for the context in which they are used (e.g. We don't rotate anything within Studio, but people commonly understand this symbol to also mean refresh.)